tests/components/ecobee/test_config_flow.py

```diff
@@ -0,0 +1,65 @@
+"""Tests for the ecobee config flow."""
+from unittest.mock import Mock, patch
+
+from homeassistant import data_entry_flow
+from homeassistant.components.ecobee import config_flow
+from homeassistant.components.ecobee.const import DATA_ECOBEE_CONFIG
+from homeassistant.const import CONF_API_KEY
+
+
+async def test_abort_if_already_setup(hass):
+    """Test we abort if ecobee is already setup."""
+    flow = config_flow.EcobeeFlowHandler()
+    flow.hass = hass
+
+    with patch.object(hass.config_entries, "async_entries", return_value=[{}]):
```

Comment:
Add a `MockConfigEntry` to hass instead.

https://github.com/home-assistant/home-assistant/blob/20e61fb41b9eb6d7b79862ee39aa202dc344dc7f/tests/common.py#L654